build/testdata/bundles/wordpress/porter.yaml

```diff
@@ -42,13 +46,34 @@ install:
         externalDatabase.database: "{{ bundle.dependencies.mysql.parameters.database-name }}"
         #externalDatabase.host: "{{ bundle.dependencies.mysql.outputs.services }}"
         externalDatabase.user: "{{ bundle.dependencies.mysql.parameters.mysql-user }}"
-        externalDatabase.password: "{{ bundle.dependencies.mysql.outputs.mysql-password }}"
+        externalDatabase.password: "{{ bundle.parameters.mysql-password }}"
         externalDatabase.port: 3306
         mariadb.enabled: false
+      outputs:
+        - name: wordpress-password
+          secret: "{{ bundle.parameters.wordpress-name }}-wordpress"
+          key: wordpress-password
+        - name: mysql-password
```

Comment:
Is this secret coming from the wordpress chart or from the mysql chart (i.e. the dependency)?